src/app/static/src/scss/partials/versions.scss

```diff
@@ -0,0 +1,3 @@
+.versions-header {
```

Comment:
Suggest dispensing with this custom class and just using Bootstrap's built in `lead` class.